templates/statistics.html.twig

```diff
@@ -0,0 +1,60 @@
+{% extends "@EasyAdmin/page/content.html.twig" %}
+
+ {% block head_stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('statistics') }}
+ {% endblock %}
+
+{% block body_javascript %}
+    {{ encore_entry_script_tags('statistics') }}
+{% endblock body_javascript %}
+
+{% block content_title %}
+    {{ 'action.statistics'|trans }}
+{% endblock %}
+
+{% block global_actions %}
+    {{ form_start(form, {'attr': {'class': 'form-inline'}}) }}
+
+        <div class="input-group">
+            <div class="input-group-prepend">
+                <span class="input-group-text">Periode</span>
```

Comment:
Add strings to translations.
Less important since there is also danish in the js.

Reply:
Partially fixed by commit 05d8a5d. Danish is still in the js.